mlrun/projects/pipelines.py

```diff
@@ -628,11 +628,13 @@ def run(
 
     @staticmethod
     def wait_for_completion(run, project=None, timeout=None, expected_statuses=None):
```

Comment:
```suggestion
    def wait_for_completion(run, project=None, timeout=None, expected_statuses=None):
```
I think it work updating the type hint as well

```
    def wait_for_completion(
        run: "_PipelineRunStatus",
        project: typing.Optional["mlrun.projects.MlrunProject"] = None,
        timeout: typing.Optional[int] = None,
        expected_statuses: list[str] = None,
    ):
```
wdyt